### PR TITLE
Update to node 10 in .travis.yml

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Upgrade to Node 10

### How to review 
Run tests make sure they pass
Create a questionnaire and make sure preview works for it

### CODE REVIEW

**Comment by rmccar** [comment]
Waiting for Node 10 to go into LTS before merging

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:8
+FROM node:10
```
Comment (samiwel) [inline Dockerfile]: I think we should be able to use `node:10-alpine` here

**Comment by rmccar** [comment]
[Review Greenkeeper PR and issues](https://trello.com/c/p2uEzQVy/631-review-greenkeeper-pr-and-issues)

File: Dockerfile (hunk 1 above)
Comment (rmccar) [reply Dockerfile]: Done!

**Pull request merged**